canonical-websites/jaas.ai
# Add Tengu cards

## Done

- Add Tengu cards to the home and store pages.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open /
- You should see a Tengu card (you might have to refresh a few times).
- Open /store
- You should see a Tengu card (you might have to refresh).

## Details

- Fixes: #137.
